### DISCUSSION TITLE
A dark palette

### DISCUSSION THREAD

**Discussion by kookma** [NicolasPetton/Notebook#6] [Ideas]
While Notebooks has its own color palettes and using those palettes it looks really great, is it possible let users to use some darker official palettes?

I tried Spartan night and Gruvbox dark. Seems all elements look fine. The side bar highlight items on mouse hover has not good contrast, but this is because of issues in official plugin colors, so may be a PR on official repo targeting BurningTreeC  help to be able use darker palettes like Nord, Gruvbox dark, Cupertino dark, Spartan night,  ...

**Comment by NicolasPetton** [comment]
It will more or less work, but Notebook defines a set of extra colors, which won't be defined in other color palettes.
I thought about trying to get rid of the colors defined in Notebook, but it's not that easy without making a compromize on the design.

**Reply by kookma** [reply]
@NicolasPetton - No problem. In the current state, Notebook is looks really great and I am sure many have switched to Notebook as default Vanilla theme is an outdated one. I will later give a try to see how can I clone beige palette and create a dark palette specific to Notebook.

**Reply by NicolasPetton** [reply]
That would be great!
